src/smm/sm-common.exports

```diff
@@ -1,3 +1,3 @@
-sm_cwa_get_mac
-sm_cwa_securize_apdu
-
+sm_cwa_get_mac
+sm_cwa_securize_apdu
+
```

Comment:
nit: Maybe the trailing newline is not needed

Reply:
I can add a commit to remove the empty lines at end of 85 files.
```
$ find . -type f -size +0 -exec gawk 'ENDFILE{if ($0 == "") print FILENAME}' {} +
./configure.ac
./src/tools/westcos-tool.c
./src/tools/dnie-tool.c
./src/common/compat_strlcat.c
./src/common/compat_getopt.c
./src/common/compat_getopt.txt
./src/common/compat_getopt.3
./src/common/simclist.h
./src/common/simclist.c
./src/tests/regression/crypt0007
./src/tests/regression/init0004
./src/tests/regression/erase
./src/tests/regression/init0001
./src/tests/regression/pin0002
[...]
$ 
```

Reply:
I can also remove spaces at the end of 1617 lines:
```
$ grep -r '[[:blank:]]$' .
./.git/packed-refs:# pack-refs with: peeled fully-peeled sorted 
./src/tools/npa-tool.c:ASN1_ITEM_TEMPLATE(ASN1_AUXILIARY_DATA_NPA_TOOL) = 
./src/tools/npa-tool.c:/** 
./src/tools/npa-tool.c: * 
./src/tools/pkcs15-crypt.c:		
./src/tools/pkcs15-crypt.c:		 * and CKU_CONTEXT_SPECIFIC login to let driver know this verify will be followed by 
./src/tools/pkcs15-crypt.c:		 * to do any special handling. 
./src/tools/npa-tool-cmdline.c:  /usr/bin/gengetopt --file-name=npa-tool-cmdline --output-dir=. 
./src/tools/npa-tool-cmdline.c:  
[...]
$ 
```

Reply:
Perhaps in a different PR?

Reply:
There is probably too much of them. We wanted to avoid mass modifications before #2017 but it is getting delayed. So I would just skip it for now. Maybe just drop it from the file you modified right now.

Reply:
Removed the final empty line from `src/smm/sm-common.exports`.